### PR TITLE
fix(cuttlefish_escript): `get` reads env var

### PR DESCRIPTION
**Pull request by 6293**

### CODE REVIEW

File: src/cuttlefish_escript.erl [hunk 1]
```diff
@@ -103,7 +103,7 @@ main(Args) ->
         describe ->
             describe(ParsedArgs, Extra);
         get ->
-            get(ParsedArgs, Extra);
+            describe(ParsedArgs, Extra, get);
```
Comment (6293) [inline src/cuttlefish_escript.erl:106]: @zmstone now I realized your suggestion is nice

**Comment by zmstone** [comment]
nice ! will have a closer look later.
enjoy the weekend Yudai.

File: test/cuttlefish_escript_test.erl [hunk 2]
```diff
@@ -138,6 +139,17 @@ get_prints() ->
...
+get_prints_env() ->
+    os:putenv("CUTTLEFISH_ENV_OVERRIDE_PREFIX", "EMQX_"),
+    os:putenv("EMQX_RING_SIZE", "16"),
+    ?capturing(begin
+                   get_("ring_size"),
+                   {ok, Stdout} = cuttlefish_test_group_leader:get_output(),
+                   ?assertEqual([["16", $\n]], Stdout)
+               end),
+    os:unsetenv("CUTTLEFISH_ENV_OVERRIDE_PREFIX"),
+    os:unsetenv("EMQX_RING_SIZE").
```
Comment (zmstone) [inline test/cuttlefish_escript_test.erl:151]:
```suggestion
    try
        ?capturing(begin
                       get_("ring_size"),
                       {ok, Stdout} = cuttlefish_test_group_leader:get_output(),
                       ?assertEqual([["16", $\n]], Stdout)
                   end)
    after
        os:unsetenv("CUTTLEFISH_ENV_OVERRIDE_PREFIX"),
        os:unsetenv("EMQX_RING_SIZE")
    end.
```

Comment (zmstone) [reply test/cuttlefish_escript_test.erl:151]: for other tests with `os:unsetenv` calls too.

**Review by zmstone — APPROVED** [review]

**Pull request merged**